src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeAzureClient.java

```diff
@@ -41,6 +41,8 @@
 
   private static final String localFileSep = systemGetProperty("file.separator");
   private static final String AZ_ENCRYPTIONDATAPROP = "encryptiondata";
+  private static final String AZ_STREAMING_INGEST_CLIENT_NAME = "ingestclientname";
+  private static final String AZ_STREAMING_INGEST_CLIENT_KEY = "ingestclientkey";
```

Comment:
Can we pull these out as static final constants somewhere and use them across all storage clients?

Reply:
I think putting constants in the interface is a bad practice, see https://stackoverflow.com/questions/2659593/what-is-the-use-of-interface-constants

Reply:
Ah I see. What about a `Constants` class? Don't we have one already that we can use?

Reply:
I don't see one, most of the constants are private and colocated in the files that are using them.

Reply:
hmmm, I can define the constants in `SnowflakeFileTransferAgent.java` and make them public so that we could pass in a Map<String, String> instead, but I'm not sure if that's better, WDYT?

Reply:
I think this is fine - if we need to extend this in the future we can do it then. Thanks for investigating!